.github/workflows/ci.yml

```diff
@@ -103,6 +104,8 @@ jobs:
       with:
         files: ${{ matrix.covname }}
         token: ${{ secrets.CODECOV_TOKEN }}
+        fail_ci_if_error: true
+        verbose: true
```

Comment:
I would prefer to fail the CI if the code coverage fails (which _should_ be rare), and if it does then we have extra verbose logs

Reply:
It's unfortunately not too rare (codecov randomly fails to accept coverage reports). I'm happy to try re-enabling this, but we may need to turn it back off in the future.